WooCommerce/Classes/Authentication/Prologue/LoginProloguePages.swift

```diff
@@ -0,0 +1,130 @@
+import UIKit
+
+// MARK: - Contents
+
+/// Details for each page of the login prologue carousel.
+///
+enum LoginProloguePageType: CaseIterable {
+    case stats
+    case orderManagement
+    case products
+    case reviews
+
+    var title: String {
+        switch self {
+        case .stats:
+            return NSLocalizedString("Track sales and high performing products",
+                                     comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .orderManagement:
+            return NSLocalizedString("Manage your store orders on the go ",
+                                     comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .products:
+            return NSLocalizedString("Edit and add new products from anywhere",
+                                     comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .reviews:
+            return NSLocalizedString("Monitor and approve your product reviews",
+                                     comment: "Caption displayed in promotional screens shown during the login flow.")
+        }
+    }
+
+    var image: UIImage {
+        switch self {
+        case .stats:
+            return UIImage.prologueAnalyticsImage
+        case .orderManagement:
+            return UIImage.prologueOrdersImage
+        case .products:
+            return UIImage.prologueProductsImage
+        case .reviews:
+            return UIImage.prologueReviewsImage
+        }
+    }
+}
+
+// MARK: - View Controller
+
+/// Simple container for each page of the login prologue carousel.
+///
+class LoginProloguePageTypeViewController: UIViewController {
+    private let stackView = UIStackView()
+    private let titleLabel = UILabel()
+    private let imageView = UIImageView()
+
+    private var pageType: LoginProloguePageType!
+
+    init(pageType: LoginProloguePageType) {
+        self.pageType = pageType
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        view = UIView()
+        view.backgroundColor = .clear
+
+        configureStackView()
+        configureImage()
+        configureTitle()
+    }
+
+    private func configureStackView() {
+        view.addSubview(stackView)
+
+        // Stack view layout
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.spacing = 40
+
+        // Reduce centerYAnchor constraint priority to ensure the bottom margin has higher priority, so stack view is fully visible on shorter devices
+        let verticalCentering = stackView.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: 103)
+        verticalCentering.priority = UILayoutPriority(999)
```

Comment:
nit: you can do `verticalCentering.priority = required - 1` in case the default required priority changes from 1000 in the future.